coordinator/gscoordinator/cluster.py

```diff
@@ -390,32 +390,30 @@ def _create_etcd(self):
 
         time.sleep(1)
 
-        # create etcd deployment
+        # create distributed etcd with 3 pods
         etcd_builder = self._gs_etcd_builder_cls(
-            name=self._etcd_name,
-            labels=labels,
-            replicas=1,
-            image_pull_policy=self._image_pull_policy,
-        )
-
-        for name in self._image_pull_secrets:
-            etcd_builder.add_image_pull_secret(name)
-
-        etcd_builder.add_etcd_container(
-            name=self._etcd_container_name,
+            name_prefix=self._etcd_name,
+            container_name=self._etcd_container_name,
             service_name=self._etcd_service_name,
             image=self._etcd_image,
             cpu=self._etcd_cpu,
             mem=self._etcd_mem,
             preemptive=self._preemptive,
+            labels=labels,
+            image_pull_policy=self._image_pull_policy,
+            num_pods=3,
```

Comment:
Maybe `num_pods` of etcd can be extracted to a configurable flag?
I thought it's better, but it's up to you.

Reply:
Expose etcd pod's number as a param named `k8s_etcd_num_pods` to client